reporting/test/helpers/scenariors.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WhoTracks.Me
+ * https://ghostery.com/whotracksme
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { decompress } from 'brotli';
+
+async function download(release, scenarior, browser) {
+  const scenartionFileName = `events_${scenarior}_${browser}.log`;
+
+  const scenariorsPath = path.join(
+    import.meta.dirname,
+    '..',
+    '..',
+    'scenariors',
+    release,
+  );
+  if (!fs.existsSync(scenariorsPath)) {
+    fs.mkdirSync(scenariorsPath, { recursive: true });
+  }
+
+  const scenariorPath = path.join(scenariorsPath, scenartionFileName);
```

Comment:
Perhaps do a replacement for the scenario typo. Looks like it is in more places.